docs/src/main/sphinx/connector/duckdb.md

```diff
@@ -11,19 +11,37 @@ DuckDB instances.
 
 ## Configuration
 
-To configure the DuckDB connector, create a catalog properties file
-in `etc/catalog` named, for example, `example.properties`, to
-mount the DuckDB connector as the `duckdb` catalog.
-Create the file with the following contents, replacing the
-connection properties as appropriate for your setup:
+The connector can query a DuckDB database. Create a catalog properties file that
+specifies the DuckDb connector by setting the `connector.name` to `duckdb`.
+
+For example, to access a database as the `example` catalog, create the file
+`etc/catalog/example.properties`. Replace the connection properties as
+appropriate for your setup:
 
 ```none
 connector.name=duckdb
-connection-url=jdbc:duckdb://<path>
-connection-user=root
```

Comment:
Username and password are actually not supported by DuckDB .. so better to remove here. Maybe even in code.